Events/DockedEvent.cs

```diff
@@ -50,9 +50,15 @@ static DockedEvent()
 
         public List<string> stationservices { get; private set; }
 
-        public DockedEvent(DateTime timestamp, string system, string station, string state, string model, Superpower allegiance, string faction, SystemState factionstate, Economy economy, Government government, decimal? distancefromstar, List<string> stationservices) : base(timestamp, NAME)
+        // Admin
```

Comment:
Comments like these don't really add anything.

Reply:
I think of it as saying that the variables below that comment are not intended to be user-facing.

Reply:
Ah I see. In that case I'd be more explicit and comment with "not user-facing"

But more generally, if we find a need to write such comments then in my experience it's a code smell. Let's not act precipitately but keep these in mind.